core/templates/pages/contributor-dashboard-page/services/translate-text.service.ts

```diff
@@ -31,12 +31,14 @@ import {
 export interface TranslatableItem {
   translation: string | string[],
   status: Status,
-  text: string | string[],
   more: boolean
   dataFormat: string,
   contentType: string,
-  interactionId?: string,
-  ruleType?: string
+  // These properties will be null when there
+  // is no interaction.
+  text: string | string[] | null,
+  interactionId: string | null,
```

Comment:
I don't quite follow this. In what circumstances is there no interaction?

Reply:
Sorry to bother you but even after trying I was not able to get the actual reason. I also don't know whom to ask this.